modules/islandora_scholar_embargo/includes/embargo.inc

```diff
@@ -485,7 +488,16 @@ EOQ;
  *   including 'obj' as the object ID, 'expiry' as the expiry date, and 'date'
  *   as the item's created or last modified date.
  */
-function islandora_scholar_embargo_get_embargoed_items_subset($offset = '1900-01-01T00:00:00.000Z', $limit = 10) {
+function islandora_scholar_embargo_get_embargoed_items_subset($offset_date = '1900-01-01T00:00:00.000Z', $offset_pid = NULL, $limit = 10) {
+  // Establish some parameters for the query.
+  $filter = is_null($offset_pid) ?
+    "FILTER(?date > \"$offset_date\"^^<http://www.w3.org/2001/XMLSchema#dateTime>)" :
```

Comment:
Could we not just avoid specifying the filter entirely, until we have a value based on data? Just use the empty string, and have things be in the proper order. We'll grab the first slice of `$limit` items, and next time we get called, we'll have the two values at which to start the next slice.